inference/benchmarks/deepseek_7b_mmlu/pytorch/evaluator.py

```diff
@@ -0,0 +1,15 @@
+import torch
+
+def evaluator(pred, y, dataloader):
+    tokenizer = dataloader.tokenizer
+    gt = y[0][0][1]
+    predict = pred[:,-1,:]
+    answer = torch.argmax(predict, dim=1)
+    answer_str = tokenizer.decode(answer)
+    valid_answers=['A','B','C','D']
+    answer_str=''.join([c for c in answer_str if c in valid_answers])
```

Comment:
need format

e.g. `yapf -i pep8 --recursive FlagPerf/`

Reply:
added format to FlagPerf/inference/benchmarks/deepseek_7b_mmlu/